build.cake

```diff
@@ -97,10 +97,10 @@ Task("Test")
 Task("PublishCalamariProjects")
    .IsDependentOn("Build")
     .Does(() => {
-        var projects = GetFiles("./source/**/Calamari.*.csproj");
+        var projects = GetFiles("./source/**/Calamari.csproj");
 		foreach(var project in projects)
         {
-            var calamariFlavour = project.GetFilenameWithoutExtension().ToString();
+            var calamariFlavour = XmlPeek(project, "Project/PropertyGroup/AssemblyName");
```

Comment:
the assembly name will always be the flavour name, even though the csproj has now been renamed